delfin/exporter/api.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2020 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from oslo_log import log
+from delfin.exporter import prometheus_exporter
+
+LOG = log.getLogger(__name__)
+
+
+class API(object):
+    def __init__(self):
+        self.pr_exporter = prometheus_exporter.PrometheusExporter()
```

Comment:
Lets intitialize based on the exporter config , see delfin.performance.exporters in setup.py